From d3d8160e273b56d0262d2570cfa6bcb76b9ea771 Mon Sep 17 00:00:00 2001
From: Thomas Moschny <thomas.moschny@gmx.de>
Date: Wed, 14 Dec 2011 20:00:53 +0100
Subject: [PATCH] Fix build with glib2 2.31.0

Starting with glib2 2.31.0, It is no longer possible to
include individual headers (like "ghash.h") -- you must
#include <glib.h>.

Signed-off-by: Thomas Moschny <thomas.moschny@gmx.de>
Signed-off-by: Uli Schlachter <psychon@znc.in>
---
 spawn.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/spawn.c b/spawn.c
index 3c8cf07..a6d6c5e 100644
--- a/spawn.c
+++ b/spawn.c
@@ -24,7 +24,7 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 
-#include <glib/gspawn.h>
+#include <glib.h>
 
 #include "spawn.h"
 #include "screen.h"
-- 
1.7.2.5

